docs/source/cpp/compute.rst

```diff
@@ -263,56 +263,56 @@ given class:
 +--------------------------+------------+--------------------+----------------+----------------------------------+
 | Function name            | Arity      | Input types        | Output type    | Matched character class          |
 +==========================+============+====================+================+==================================+
-| ascii_isalnum            | Unary      | String-like        | Boolean        | Alphanumeric ASCII               |
+| ascii_is_alnum            | Unary      | String-like        | Boolean        | Alphanumeric ASCII               |
```

Comment:
Ok, you need to keep the table markup correct, so be careful to align everything.

In general, _please_ rerender the doc when making changes to it. Too often some edits are made to the doc without even checking that the markup is ok (i.e. no additional warnings are shown when building it).

Reply:
Is there a way I can render just this file? In the past when I've tried to render all of the sphinx docs, it's been such a PITA, partly because I have to build pyarrow, partly because of my case-insensitive file system.

Reply:
Can you open an issue for the case-insensitive issue?

Reply:
There was an issue and it was decided that the solution was to build the docs for the website only on a case-sensitive file system 🤷

Reply:
At worse you can take a look at the Github rendering (click "View this file"). Unfortunately, it doesn't show any errors explicitly, but apparently it doesn't display the misformatted tables.

Reply:
If you push changes to this PR, I'll try rendering locally and tell you.

Reply:
Ok, rendering worked fine here.

Reply:
@nealrichardson What was the issue number?

Reply:
https://issues.apache.org/jira/browse/ARROW-8078 was the one I was thinking of

Reply:
Ah, thank you. Well, indeed there's not much that we can do (except break our public API :-)).